build-scripts/components/runc/build.sh

```diff
@@ -5,6 +5,14 @@ VERSION="${2}"
 export INSTALL="${1}/bin"
 mkdir -p "${INSTALL}"
 
+# Pin runc to Go 1.21: https://github.com/opencontainers/runc/issues/4233
```

Comment:
nit: should we track that e.g in a card? or add a TODO to remove this once the runc issue is resolved.